catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/data/impl/MetacardImplTest.java

```diff
@@ -142,14 +143,14 @@ public void testMetacardImpl() {
     assertEquals(null, mi.getContentTypeName());
     assertEquals(null, mi.getContentTypeNamespace());
     assertEquals(null, mi.getContentTypeVersion());
-    assertEquals(null, mi.getCreatedDate());
+    assertEquals(null, mi.getAttribute(Core.CREATED).getValue());
```

Comment:
why do these need to change?  the constant was updated https://github.com/codice/ddf/pull/4608/files#diff-c955ba16e1794205a33bc834339c7aa7R232

Reply:
I wanted to remove the references to these methods because they are now ambiguous with the catalog taxonomy. The javadoc claims
>  Returns the date/time this {@link Metacard} was created.
However, the attribute that the method actually returns instead refers to the resource created date. This is due to the older taxonomy of "created" referring to metacard dates, while in the new taxonomy, "created" refers to the resource date, and instead "metacard.created" refers to the metacard date. 

Basically, I marked it as deprecated and removed its usage because it doesn't do what it was originally designed to do. We could update it to instead return the new "metacard.modified" attribute, but I'm not sure what downstream behavior is expecting at this point. Every usage out there is either ignoring the javadoc, or is not doing what was intended.